TeamCode/src/main/java/org/firstinspires/ftc/teamcode/AprilTagDetectionFrameProcessor.java

```diff
@@ -32,42 +33,46 @@
 import org.opencv.imgproc.Imgproc;
 import org.openftc.apriltag.AprilTagDetection;
 import org.openftc.apriltag.AprilTagDetectorJNI;
-import org.openftc.easyopencv.OpenCvPipeline;
 
 import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A FrameProcessor that detects April tags.
+ *
+ * Adapted with permission from the sample code developed by the OpenFTC Team, 2021.
+ */
+public class AprilTagDetectionFrameProcessor implements WebCam.FrameProcessor {
```

Comment:
This is the key change: make it implement `WebCam.FrameProcessor` instead of `OpenCvPipeline`